tests/foreman/ui/test_contenthost.py

```diff
@@ -1250,3 +1260,59 @@ def test_syspurpose_mismatched(session, vm_module_streams):
             'details'
         ]
         assert details['system_purpose_status'] == "Mismatched"
+
+
+@tier4
+def test_pagination_multiple_hosts_multiple_pages(session, module_host_template):
+    """Create hosts to fill more than one page, sort on OS, check pagination.
+
+    Search for hosts based on operating system and assert that more than one page
+    is reported to exist and that more than one page can be accessed. Make some
+    additonal aserts to ensure the pagination widget is working as expected.
+
+    To avoid requiring more than 20 fakes hosts to overcome default page setting of 20,
+    this test will set a new per_page default of 4, using URL method rather than the
```

Comment:
```suggestion
    this test will set a new per_page default of 3, using URL method rather than the
```
I guess it should be 3.

Reply:
Right, thanks for spotting that. The test has changed a few times and I missed that. I am now chaning it to use refer to the new_per_page_setting variable.